src/Persistence/PaPersistenceException.cs

```diff
@@ -9,30 +9,30 @@
 namespace Microsoft.PowerPlatform.PowerApps.Persistence;
 
 [Serializable]
-public class PersistenceException : Exception
+public class PaPersistenceException : Exception
```

Comment:
Can we use PersistenceException like it was before?

Reply:
Name collides with exceptions in DocServer IIRC

Reply:
Namespace already has "PowerApps" in it. I don't think name should have extra "Pa" = "PowerApps" again in it

Reply:
This rename is to remove ambiguity with another exception that exists already in the doc svr code.

Reply:
Ideally ambiguity is resolved using namespaces

Reply:
in this codebase:
`Microsoft.PowerPlatform.PowerApps.Persistence.PersistenceException`

in doc server codebase:
`Microsoft.AppMagic.Authoring.Persistence.PersistenceException`

In many places in the doc server codebase, both of these namespaces are imported via `using` statements.
When needing to reference these types, we must resolve the ambiguity. Yes, it's totally possible to use full names in place or a namespace/type alias statements to help resolve the ambiguity. Yet it's a very clumbsy thing to deal with, also, if people aren't aware of this ambiguity, the error that shows up deals doesn't indicate an ambiguity sometimes, but just says the ctor parameters is wrong. This acutally points to the wrong type being referenced.

This change is a simple pragmatic change which removes all ambiguity or need to do type resolution gymnastics to get the right type.

The result is:
- simpler code
- less ambiguity
- less developer frustration
- happier devs :D

Reply:
I've renamed it to `PersistenceLibraryException` per result of meeting. Thanks @tehcrashxor for the new name suggestion!